src/overlay/OverlayMetrics.cpp

```diff
@@ -92,6 +92,14 @@ OverlayMetrics::OverlayMetrics(Application& app)
     , mOutboundQueueDelayDemand(
           app.getMetrics().NewTimer({"overlay", "outbound-queue", "demand"}))
 
+    , mOutboundQueueDropSCP(app.getMetrics().NewMeter(
```

Comment:
forgot to update docs

Reply:
Updated. (covered under `overlay.outbound-queue.<X>`)

Reply:
Split it into those two.